Makefile

```diff
@@ -17,17 +17,19 @@ run-camera-simulator:
 	./camera-simulator/camera-simulator.sh
 
 clean:
-	if [ -z $$(docker ps  --filter="name=automated-self-checkout" -q -a) ]; then\
+	CONTAINER_ASC=$$(docker ps  --filter="name=automated-self-checkout" -q -a);\
+	if [ -z $$CONTAINER_ASC ]; then\
 		 echo "nothing to clean up";\
 	else\
-		docker rm $$(docker ps  --filter="name=automated-self-checkout" -q -a) -f;\
+		docker rm $$CONTAINER_ASC -f;\
```

Comment:
[use map file to avoid array splitting: see this](https://github.com/intel-retail/automated-self-checkout/blob/1.5.0/benchmark-scripts/benchmark.sh#L299)